shared/interop/generate_genesis_state.go

```diff
@@ -74,6 +92,28 @@ func GenerateDepositsFromData(depositDataItems []*ethpb.Deposit_Data, trie *trie
 
 // DepositDataFromKeys generates a list of deposit data items from a set of BLS validator keys.
 func DepositDataFromKeys(privKeys []*bls.SecretKey, pubKeys []*bls.PublicKey) ([]*ethpb.Deposit_Data, [][]byte, error) {
+	type depositData struct {
```

Comment:
I'd like to do this (or something like this).  Given that the sequential code is now in a private function `depositDataFromKeys()` how do you think I should go about this?  My problem is that the tests are in `interop_test` so cannot access the sequential code.

Note that I do have tests for scatter's correct operation in `shared/mputil` where the values are calculated so I can easily do the check, although not for a situation as complex as creating the genesis state.

Reply:
The way I decided to go was add a separate test file `generate_deposits_internal_test.go` that is in the `interop` package rather than `interop_test`.  This allows me to access the SP-version of the functions and hence do comparisons.